tests/providers/amazon/aws/log/test_cloudwatch_task_handler.py

```diff
@@ -68,7 +68,7 @@ def setup_tests(self, create_log_template, tmp_path_factory, session):
         task_id = "task_for_testing_cloudwatch_log_handler"
         self.dag = DAG(dag_id=dag_id, schedule=None, start_date=date)
         task = EmptyOperator(task_id=task_id, dag=self.dag)
-        dag_run = DagRun(dag_id=self.dag.dag_id, execution_date=date, run_id="test", run_type="scheduled")
+        dag_run = DagRun(dag_id=self.dag.dag_id, logical_date=date, run_id="test", run_type="scheduled")
```

Comment:
Hmm, we’ll need to do something with this since providers need to still support 2.x. Without breaking anything, I think the best thing to do would be to keep `execution_date` on DagRun’s `__init__` for now, and let providers use it until they drop Airflow 2.x support.